### DISCUSSION TITLE
Welcome to DataKit Discussions!

### DISCUSSION THREAD

**Discussion by zackkatz** [UseDataKit/SDK#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JurriaanK** [comment]
Hi Team,
Jurriaan here from [GravityWP](https://gravitywp.com).
We spoke at WCEU about this idea and I just had a look. Just wanted to say this is simply great! I really like the code structure and the built in quality assurance with Automated Tests, Coding Standards, PHPStan, etc. Great job so far, can't wait to see were this is going.
We will definitely try this SDK in one of our next [Aiwos](https://aiwos.com/en) projects. We will let you know how it goes!
